documentation/homework/week4/src/stores/InitialStore.js

```diff
@@ -0,0 +1,102 @@
+import { observable, action } from 'mobx'
+
+// Setting our localStorage
+import { locals } from '../utils'
+
+// set the default state of the items Object
+const default_item_state = {
+  id: 0,
+  title: '',
+  deadline: '',
+  done: false,
+  Edit: false,
+}
+
+class Counter {
```

Comment:
First, you forgot to rename the class when copy pasting! Second, why are you calling this store `InitialStore`?

Reply:
Well First i didn't copy paste! and i was just following the Counter lesson making things simple.
second I couldn't find a name 🙃